### ISSUE TITLE
de l'élément `<tag>`

### ISSUE THREAD

**Issue by borisschapira** [cleverage/accessibility-steps#27]
Plusieurs possibilités, votez pour qu'on essaie d'homogénéiser.
Exemple : "de l'élément `<tag>`"

**Comment by borisschapira** [comment]
> of the `<tag>` element

**Comment by borisschapira** [comment]
> of the element `<tag>`

**Comment by borisschapira** [comment]
>  of the `tag` tag

**Comment by borisschapira** [comment]
>  of the tag `<tag>`

**Comment by ryuran** [comment]
I prefere "element" for:

> `<title>` element in the `<head>` element

And "tag" for:

> attribute `lang` of the `<html>` tag

**Comment by nhoizey** [comment]
Non, mais s'il y a des chevrons, c'est un tag, s'il n'y en a pas c'est un élément
